ui/src/EmailTemplateList.js

```diff
@@ -188,19 +190,24 @@ class EmailTemplateList extends Component {
       }
     }
 
+
+
     if (!this.state.template.trim()) {
       this.setState({ templateValidationError: true });
       failedValidation = true;
     } else {
       try {
         const parsedJson = JSON.parse(this.state.template);
-        if (!Array.isArray(parsedJson)) {
+        const hasDuplicateTemplateColumns = new Set(parsedJson).size !== parsedJson.length;
+        if (!Array.isArray(parsedJson) || hasDuplicateTemplateColumns) {
           this.setState({ templateValidationError: true });
           failedValidation = true;
+          this.setState({ templateValidationErrorMessage: "email template must be JSON array with one or more unique elements" });
```

Comment:
Not seeing this validation message. I'm seeing `Error Creating SMSTemplate` instead